rvm.sh

```diff
@@ -0,0 +1,939 @@
+#!/usr/bin/env bash
```

Comment:
If you need this file on your local machine, please add it to `.gitignore` file. If you do not need this file, please remove it

Reply:
Removed the file